pkg/metrics/metrics.go

```diff
@@ -33,8 +33,10 @@ const (
 	Namespace = "upgradeoperator"
 	Subsystem = "upgrade"
 
-	StateLabel   = "state"
-	VersionLabel = "version"
+	StateLabel            = "state"
+	VersionLabel          = "version"
+	PrecedingVersionLabel = "preceding_version"
+	MinorUpgradeLabel     = "minor_upgrade"
```

Comment:
Will it be more explicit to use z-stream upgrade here instead of minor version upgrade?

Reply:
changed to stream = y/z